<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B9Layout</class>
 <widget class="QMainWindow" name="B9Layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="layoutBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Layout</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Model List</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QPushButton" name="AddButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemoveButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="ModelList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="layoutMode">
             <enum>QListView::SinglePass</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ModelTranslationBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Model Information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Size(mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="modelsizex">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="modelsizey">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="modelsizez">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position(mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="posx"/>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="posy"/>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="posz"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rotx"/>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="roty"/>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rotz"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Scaling Factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="scalex"/>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="scaley"/>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="scalez"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="scalelock">
            <property name="text">
             <string>Lock Aspect Ratio</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ApplyButton">
            <property name="text">
             <string>Apply Changes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printButton">
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="planingbox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>3D View</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="WorldViewContext"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainmenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Model"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCenter_View"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSelection"/>
    <addaction name="actionMove"/>
    <addaction name="actionRotate"/>
    <addaction name="actionScale"/>
    <addaction name="separator"/>
    <addaction name="actionDrop_To_Floor"/>
    <addaction name="actionDuplicate"/>
   </widget>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>Developer</string>
    </property>
    <addaction name="actionDebug_Slices"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuDeveloper"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="../b9edit/b9edit.qrc">
     <normaloff>:/B9Edit/icons/new.png</normaloff>:/B9Edit/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="../b9edit/b9edit.qrc">
     <normaloff>:/B9Edit/icons/open.png</normaloff>:/B9Edit/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../b9edit/b9edit.qrc">
     <normaloff>:/B9Edit/icons/save.png</normaloff>:/B9Edit/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCenter_View">
   <property name="text">
    <string>Center View</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionDrop_To_Floor">
   <property name="text">
    <string>Drop To Floor</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionDebug_Slices">
   <property name="text">
    <string>Debug Slices</string>
   </property>
  </action>
  <action name="actionAdd_Model">
   <property name="text">
    <string>Add Model</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../b9edit/b9edit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNew_Project</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>New()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Project</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>SaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Model</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>AddModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>DeleteSelectedInstances()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelection</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>SetToolPointer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMove</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>SetToolMove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>SetToolRotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrop_To_Floor</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>DropSelectionToFloor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebug_Slices</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>OpenDebugWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDuplicate</sender>
   <signal>activated()</signal>
   <receiver>B9Layout</receiver>
   <slot>DuplicateSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printButton</sender>
   <signal>clicked()</signal>
   <receiver>B9Layout</receiver>
   <slot>SliceWorld()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>B9Layout</receiver>
   <slot>AddModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemoveButton</sender>
   <signal>clicked()</signal>
   <receiver>B9Layout</receiver>
   <slot>DeleteSelectedInstances()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posx</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posy</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posz</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotx</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roty</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotz</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalex</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaley</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalez</sender>
   <signal>returnPressed()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ApplyButton</sender>
   <signal>clicked()</signal>
   <receiver>B9Layout</receiver>
   <slot>PushTranslations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>New()</slot>
  <slot>Open()</slot>
  <slot>Save()</slot>
  <slot>SaveAs()</slot>
  <slot>SetXYPixelSize(QString)</slot>
  <slot>SetZLayerThickness(QString)</slot>
  <slot>LinkDetail()</slot>
  <slot>SetProjectorX(QString)</slot>
  <slot>SetProjectorY(QString)</slot>
  <slot>SetProjectorFromIndx(int)</slot>
  <slot>ProjectorCustomSwitch()</slot>
  <slot>AddModel()</slot>
  <slot>DeleteSelectedInstances()</slot>
  <slot>SetToolPointer()</slot>
  <slot>SetToolMove()</slot>
  <slot>SetToolRotate()</slot>
  <slot>SliceWorld()</slot>
  <slot>LockScale(bool)</slot>
  <slot>DropSelectionToFloor()</slot>
  <slot>DuplicateSelection()</slot>
  <slot>PushTranslations()</slot>
  <slot>OpenDebugWindow()</slot>
 </slots>
</ui>
